soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -1261,10 +1261,10 @@ void GenerateItemPool() {
     PlaceVanillaDekuScrubItems(ctx->GetOption(RSK_SHUFFLE_SCRUBS).Is(RO_SCRUBS_OFF));
   }
 
-  bool overworldFreeStandingActive = ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_FREESTANDING_OVERWORLD) ||
-                             ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_FREESTANDING_ALL);
-  bool dungeonFreeStandingActive = ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_FREESTANDING_DUNGEONS) ||
-                           ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_FREESTANDING_ALL);
+  bool overworldFreeStandingActive = ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_SHUFFLE_FREESTANDING_OVERWORLD) ||
+                             ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_SHUFFLE_FREESTANDING_ALL);
+  bool dungeonFreeStandingActive = ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_SHUFFLE_FREESTANDING_DUNGEONS) ||
+                           ctx->GetOption(RSK_SHUFFLE_FREESTANDING).Is(RO_SHUFFLE_FREESTANDING_ALL);
   if (overworldFreeStandingActive || dungeonFreeStandingActive) {
     PlaceItemsForType(RCTYPE_FREESTANDING, overworldFreeStandingActive, dungeonFreeStandingActive, true);
```

Comment:
can the last bool parameter be removed? it places vanilla items into all the checks when disabled. If it should be kept then the conditional around this call needs to be removed

Reply:
After further inspection there's something else wrong here. First of all freestanding is trying to mimic pots, but doesn't have the separation of checks inside DungeonInfo to use it properly, but there might be other issues here too. I wanted to wait to do all that but I guess I have no choice to look into it now then.

Reply:
Oh never mind, removing the checks *does* work because it's placing vanilla locations via PlaceItemsForType. But at this point I might as well look into not including the locations at all.